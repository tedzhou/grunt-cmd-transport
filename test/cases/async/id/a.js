define(function(require){
	var $ = require('jquery');
	var b = require('id/b');
	require.async('id/c', function(c){
		return c;
	});
})