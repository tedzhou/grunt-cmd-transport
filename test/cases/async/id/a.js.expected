define("id/a", [ "jquery", "id/b", "id/f", "id/e" ], function(require) {
    var $ = require("jquery");
    var b = require("id/b");
    require.async([ "id/c", "id/d", "id/e" ], function() {
        (function(c) {
            return c;
        }).apply(this, [].slice.call(arguments, 0, 1));
    });
});